tests/FSharpPlus.Tests/General.fs

```diff
@@ -1162,6 +1163,29 @@ module Applicative =
         Assert.AreEqual ([113; 213; 123; 223; 114; 214; 124; 224], testVal)
         Assert.IsInstanceOf<Option<list<int>>> (Some testVal)
 
+        let testLTE1 = (Some 1) .<=. (Some 2)
```

Comment:
Do we really need enclosing parens here?
I think the operator alone should be enough.